tools/src/test/java/com/nedap/archie/adlparser/RMPathQueryTest.java

```diff
@@ -27,7 +33,7 @@ public class RMPathQueryTest {
 
 
     private TestUtil testUtil;
-    private Archetype archetype;
+    private Archetype archetype, archetype_specialised, archetype_specialised_twice;
```

Comment:
I think it is better to use local variables for `archetype_specialised` and `archetype_specialised_twice` as they are only used in individual methods and not preloaded in `setup`.